main.js

```diff
@@ -0,0 +1,1146 @@
+					},
+				],
+				callback: async (event) => {
+					const path = await this.parseVariablesInString(event.options.path)
+					const int = await this.parseVariablesInString(event.options.int)
+
+					sendOscMessage(path, [
+						{
+							type: 'i',
+							value: parseInt(int),
+						},
+					])
+				},
+			},
+			send_ppt18: {
+				name: 'PPT 18',
+				options: [
+					{
+						type: 'textinput',
+						label: 'COMANDO',
+						id: 'path',
+						default: '/abrir18',
+						useVariables: true,
+					},
+					{
+						type: 'textinput',
+						label: 'Value',
+						id: 'int',
+						default: 1,
+						regex: Regex.SIGNED_NUMBER,
+						useVariables: true,
+					},
+				],
+				callback: async (event) => {
+					const path = await this.parseVariablesInString(event.options.path)
+					const int = await this.parseVariablesInString(event.options.int)
+
+					sendOscMessage(path, [
+						{
+							type: 'i',
+							value: parseInt(int),
+						},
+					])
+				},
+			},
+			send_ppt19: {
+				name: 'PPT 19',
+				options: [
+					{
+						type: 'textinput',
+						label: 'COMANDO',
+						id: 'path',
+						default: '/abrir19',
+						useVariables: true,
+					},
+					{
+						type: 'textinput',
+						label: 'Value',
+						id: 'int',
+						default: 1,
+						regex: Regex.SIGNED_NUMBER,
+						useVariables: true,
+					},
+				],
+				callback: async (event) => {
+					const path = await this.parseVariablesInString(event.options.path)
+					const int = await this.parseVariablesInString(event.options.int)
+
+					sendOscMessage(path, [
+						{
+							type: 'i',
+							value: parseInt(int),
+						},
+					])
+				},
+			},
+			send_ppt20: {
+				name: 'PPT 20',
+				options: [
+					{
+						type: 'textinput',
+						label: 'COMANDO',
+						id: 'path',
+						default: '/abrir20',
+						useVariables: true,
+					},
+					{
+						type: 'textinput',
+						label: 'Value',
+						id: 'int',
+						default: 1,
+						regex: Regex.SIGNED_NUMBER,
+						useVariables: true,
+					},
+				],
+				callback: async (event) => {
+					const path = await this.parseVariablesInString(event.options.path)
+					const int = await this.parseVariablesInString(event.options.int)
+
+					sendOscMessage(path, [
+						{
+							type: 'i',
+							value: parseInt(int),
+						},
+					])
+				},
+			},
+			send_esc: {
+				name: 'Fechar Modo Apresentacao',
+				options: [
+					{
+						type: 'textinput',
+						label: 'COMANDO',
+						id: 'path',
+						default: '/esc',
+						useVariables: true,
+					},
+					{
+						type: 'textinput',
+						label: 'Value',
+						id: 'int',
+						default: 1,
+						regex: Regex.SIGNED_NUMBER,
+						useVariables: true,
+					},
+				],
+				callback: async (event) => {
+					const path = await this.parseVariablesInString(event.options.path)
+					const int = await this.parseVariablesInString(event.options.int)
+
+					sendOscMessage(path, [
+						{
+							type: 'i',
+							value: parseInt(int),
```

Comment:
What happens when the user enters `-149588357.2`? Or `this is not a number`? Regex on option does not validate input, it just gives a visual hint.